transbank-sdk-dotnet-core-rest-example/Views/transaccion_completa_mall_deferred/commit.cshtml

```diff
@@ -72,7 +72,7 @@
         Ya puedes mostrar al usuario una página de éxito de la transacción.
     </div><br>
 
-    @{Html.RenderPartial("~/Views/deferred_common/deferred-options.cshtml");}
+    @{Html.RenderPartial("~/Views/deferred_common/deferred-optionsTx.cshtml");}
```

Comment:
use deferred-options-tx instead deferred-optionsTx

Reply:
the name was changed to deferred-options-tx